### ISSUE TITLE
[w/c 02/09/2024] RIKEN: RB2 PSU Replacement - connect controllers and configure IOCs

### ISSUE THREAD

**Issue by davidkeymer** [ISISComputingGroup/IBEX#8477]
The RIKEN RB2 PSU is being replaced during this shutdown, with the work (from our point of view) due to be complete in the week commencing 02/09/2024.  This task can only be started once the installation is complete and power restored,

### Acceptance criteria:
- The two Danfysik PSU controllers are connected to the appropriate MOXA NPort
- The IOCs `DFKPS_33` and `DFKPS_34` are re-configured by editing macros accordingly (see `globals.txt` file in NDXRIKENFE configuration area).
- Existing IBEX blocks function as before (should not need editing as existing IOCs & PVs reused)

### Notes
- Cable runs exist from the existing PSU to the MOXA NPort in the RIKEN area - reuse these for the new PSU
- Mapping from MOXA NPort sockets to PSU and IOCs document [here](https://stfc365.sharepoint.com/:x:/r/sites/ISISExperimentControls/ICP%20Discussions/RIKEN%20FE/RIKENFE%20MOXA%20NPort%20Devices%20and%20IOCs.xlsx?d=wed7ad8b945c24f9e9987c86f77fad960&csf=1&web=1&e=avcs5o)
- Check that the MOXA NPort does _NOT_ have any RS485 termination enabled for the particular ports
- The macros to be changed in particular are those for the Danfysik PSU type and the values for `MAX_RAW_SETPOINT` in ppm (these will need to obtained from the DC PSU group - RH and/or TS).

### Time in Planning
~00:13 08/08/24

**Comment by Tom-Willemsen** [comment]
Currently impeded due to hardware faults with RB2

**Comment by Tom-Willemsen** [comment]
Configured on RIKENFE last week, JL notified about what blocks are available (I think he has since renamed a couple of them to his liking), supplies are working from an IBEX perspective (they are in use at the moment, with no issues reported since end of last week).

Needed to do a hotfix on RIKENFE to use `WA` rather than `DA 0` to set currents, see https://github.com/ISISComputingGroup/IBEX/issues/8502 for details, the supplies would likely still work without that hotfix, but have implemented the recommendation as per the note linked in that issue. Spun out into a separate issue as it'll want tests & emulators etc updating too.

To review this ticket:
- Point a local client at RIKENFE
- Check blocks are available for RB2_1 and RB2_2 (CURR, VOLT, POL, POWER)
- Check the device screen indicates the PSUs are communicating (and likely in use)
